charts/httpd/templates/httpdRestart.yaml

```diff
@@ -0,0 +1,64 @@
+{{- if .Values.httpdRestart.enable }}
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: Role
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+rules:
+  - apiGroups: ["apps", "extensions"]
+    resources: ["deployments"]
+    resourceNames: [{{ include "httpd.fullname" . }}]
+    verbs: ["get", "patch", "list", "watch"]
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ include "httpd.fullname" . }}-restart-deployment
+subjects:
+  - kind: ServiceAccount
+    name: {{ include "httpd.fullname" . }}-restart-deployment
+---
+apiVersion: batch/v1
+kind: CronJob
+metadata:
+  name: {{ include "httpd.fullname" . }}-restart-cronjob
+  labels:
+{{ include "httpd.labels" . | indent 4 }}
+spec:
+  successfulJobsHistoryLimit: 1
+  failedJobsHistoryLimit: 2
```

Comment:
```suggestion
```

You mentioned we could eventually fine tune later: in that case let's stick with the Kubernetes default and avoid surprises

> we need to keep in mind that we may need to provide a cleanup for both [cronjobs](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#jobs-history-limits) and [jobs](https://kubernetes.io/docs/concepts/workloads/controllers/job/#clean-up-finished-jobs-automatically) through the .spec.successfulJobsHistoryLimit and .spec.failedJobsHistoryLimit fields for the first and the .spec.ttlSecondsAfterFinished field for the second